MoreLinq/NoConflict.ToDataTable.cs

```diff
@@ -0,0 +1,89 @@
+namespace MoreLinq.NoConflict
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Data;
+    using System.Linq.Expressions;
+
+    /// <summary><c>ToDataTable</c> extension.</summary>
+    public static partial class ToDataTableExtension
+    {
+        /// <summary>
+        /// Appends elements in the sequence as rows of a given <see cref = "DataTable"/> 
```

Comment:
I noticed those too. Not sure why it's happening. Could be the result of formatting applied by [`NormalizeWhitespace()`](https://github.com/morelinq/MoreLINQ/pull/235/files#diff-bdf255a032eb2d9447b419908ff8f510R127).